storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1185,6 +1185,7 @@ bool Redo_Log_Data_Manager::init() {
         xtrabackup_register_redo_log_consumer = false;
         return (false);
       }
+      xb_mysql_query(redo_log_consumer_cnx, "SET sql_mode=''", false, true);
```

Comment:
what about not usin double quotes?

Reply:
it's either that or enforce sql_mode. I thought about enforcing SQL mode so we do not get surprises on any other variation, or mix of sql_modes that are not compatible with the consumer.

Reply:
I would only change the sql_mode everywhere with more detailed research on all the queries on those connections. Consumer only executes three queries: register, advance, and deregister consumer.

Reply:
yeah, now we are back to old times. GCA between 8.0 and trunk and merge it into two branches

Reply:
btw, this branch was already created based on the GCA point between 8.0 and trunk